build/tasks.ts

```diff
@@ -235,6 +235,11 @@ const ensureClean = register("vscode:clean", async (runner) => {
 
 	const status = await runner.execute("git", ["status", "--porcelain"]);
 	if (status.stdout.trim() !== "") {
+
+		// this is an uggly workaround - inside docker git clean throws an error message because of this directory
+		// related to git modules.
+		await runner.execute("rm", ["-rf", "node_modules/spdlog"]);
```

Comment:
I've never ran into an error with this. Can you describe in more detail what you did and what happened?

Reply:
the git clean command throws an error, that a submodule is not git repository.
I didn't get the error message on my mac - but inside docker yes.

Give me a minute and I try to reproduce the issue.

Reply:
reverted. Maybe it was an earlier version of node - or the yarn - installed by npm (yakk). 
Now its working without the workaround